pages/visit/events-exhibits/_slug.vue

```diff
@@ -2,63 +2,124 @@
     <section class="page-event-detail">
         <!-- this template will pick the section page component based on typehandle -->
         <nav-breadcrumb
-            v-if="allEvents[0]"
-            :title="allEvents[0].title"
+            :title="page.slug"
+            :to="page.to"
+            :parent-title="page.sectionHandle"
         />
 
-        <masthead-secondary
+        <!-- this wasn't in the design, so I just commented it out -->
+        <!-- <masthead-secondary
             title="Exhibits & Upcoming Events"
             text="Browse upcoming remote events and online exhibits."
-        >
-            <!-- TODO Add SearchGenric here when complete -->
-            <!-- search-generic
-                search-type="about"
-                class="generic-search"
-            />-->
-            <!-- :filters="searchFilters.filters"
-                :view-modes="searchFilters.views"
-                @view-mode-change="viewModeChanger" -->
-        </masthead-secondary>
-
-        <p v-if="$fetchState.pending" />
-        <p v-else-if="$fetchState.error">
-            An error occurred :(
-        </p>
-        <header-sticky
+        /> -->
+        
+        <!-- <header-sticky
             class="sticky-header"
             :primary-items="primaryItems"
             :secondary-items="secondaryItems"
-        />
+        /> -->
+
         <banner-text
-            v-if="formData"
-            category="Event"
-            title="Curabitur Tortor Pellentesque"
-            text="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec accumsan, metus in aliquet venenatis, mi lectus placerat leo, congue gravida mi quam sit amet neque."
-            button-text="Curabitur"
-            :register-event="true"
+            v-if="page && (!page.image[0].image[0] || page.image[0].image[0].length == 0)"
+            :title="page.title"
+            :locations="page.associatedLocations"
+            :start-date="page.date.startTime"
+            :category="page.eventType.title"
+            to="https://jira.library.ucla.edu/browse/APPS-2015"
         />
 
+        <!-- if theres an image -->
+        <section-wrapper
+            v-if="page.image[0].image[0]"
+            class="section-banner"
+        >
+            <banner-header
+                :image="page.image[0].image[0]"
+                :title="page.title"
+                :locations="page.associatedLocations"
+                :start-date="page.date.startTime"
+                :category="page.eventType.title"
+                to="https://jira.library.ucla.edu/browse/APPS-2015"
+                :align-right="true"
+                prompt="View exhibit"
```

Comment:
prompt pro is used on both banner-text and Banner-header when  to prop exists and pass :prompt="More Details"